### DISCUSSION TITLE
CSS 파일을 불러오지 못합니다.

### DISCUSSION THREAD

**Discussion by jinyongp** [pronist/hELLO#75] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: macOS Monterey (M1 2020)
- 브라우저: brave (chrome 또한 발생)
- 버전: V1.35.100
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): 3.6.1

**설명**
버그가 어떻게 발생하는지 알려주세요. 대략적인 버그의 설명을 적어주시면 됩니다.
[e.g. 코드 하이라이팅이 동작하지 않습니다.]

스킨 자체가 이상하게 보입니다. preview, 배포 두 곳 동일하게 발생합니다. 

사이드 바 열었을 때 전체 화면이 어두운 반투명 효과가 적용됩니다. (의도하신 건진 모르겠지만 class에 overlay가 붙어서 스타일이 들어가세요.)

블로그에 기본적으로 사이드가 오픈되어 있던데 고정은 어떻게 하는지 잘 모르겠습니다. 그리고 사이드바가 투명 	배경이 되어 있어 뒤가 그대로 보입니다.

다크모드 버튼을 누르면 아래 올린 에러가 발생합니다.

**발생 과정**
버그가 어떤 과정을 통해 발생하는지 구체적으로 적어주세요.
[e.g. 테마 변경 버튼을 통해 화이트 모드로 변경시 코드 하이라이팅의 색상이 변하지 않습니다. ...]

README.md 설치 과정을 그대로 따랐습니다.

**스크린샷 및 링크**
버그가 발생한 상황을 가능하다면 스크린샷을 첨부해주세요. 또한 버그가 발생한 모습이 있는 포스트의 주소를 첨부해주시는 것도 아주 좋습니다!

![image](https://user-images.githubusercontent.com/21158729/153000124-b9f796ac-e43f-4c57-9234-858d5c77139c.png)

**Comment by pronist** [comment]
@jinyongp 스킨옵션을 봐야할 듯합니다. 스킨옵션을 보여주실 것을 요청합니다. 스킨에서 사용하는 스타일은 스킨옵션에 따라 결정됩니다.

**Comment by pronist** [comment]
@jinyongp 추가적으로 [어떤 기능을 제공합니까?](https://pronist.dev/5?category=844785#%EC%--%B-%EB%--%A-%--%EA%B-%B-%EB%-A%A-%EC%-D%--%--%EC%A-%-C%EA%B-%B-%ED%--%A-%EB%-B%--%EA%B-%-C%-F) 를 참고하여 어떤 스킨 옵션이 있는지 참고해주시고, 만약 스킨 옵션이 배포 페이지에서 설명하는 것과 다르게 나온다면 ```index.xml``` 을 바르게 업로드하였는지 점검해주시기 바랍니다.

**Comment by jinyongp** [comment]
스킨 옵션은 [환경설정](https://tidory.com/docs/configuration/)에서 나온 `skinSettings` 말씀하시는건가요? `ts_session`하고 `url`만 수정하고 다른 건 따로 추가하거나 손대진 않았습니다.

**Comment by pronist** [comment]
@jinyongp 지금 어떤코드를 보고계신건가요? 스킨 배포페이지 - 다운로드에서 배포파일을 다운 받고 적용하신거 맞으신가요?

**Comment by pronist** [comment]
@jinyongp 아니면 스킨 적용문제가 아니라, 개발관련 문의시라면 ```index.xml``` 에 설정된 기본 값을 살펴볼 필요가 있습니다. ```window.H.skinOptions``` 에서 값을 제대로 불러오는지 살펴보아야 합니다.

**Comment by jinyongp** [comment, marked as answer]
제가 정신이 없었나봅니다.. 번거롭게 해서 죄송해요 ㅜㅜ 이용만 하려고 한건데 습관대로 레포를 받아버렸네요... 잘 사용하겠습니다. 좋은 스킨 만들어주셔서 감사합니다.
